collectd/ping.sls

```diff
@@ -3,7 +3,12 @@
 include:
   - collectd
 
-liboping0:
+{% set ping_pkg = {
+  'RedHat': 'collectd-ping',
+  'Debian': 'liboping0'
+}.get(grains['os_family']) %}
+
+{{ ping_pkg }}:
```

Comment:
Probably best to change this ID to something that has less chance of clashing and using name/pkgs (to be more in line with what other formulas do). Then you can change the require to just the ID.